Mathlib/Analysis/Complex/Basic.lean

```diff
@@ -422,6 +422,11 @@ noncomputable instance : IsROrC ℂ where
   conj_I_ax := conj_I
   norm_sq_eq_def_ax z := (normSq_eq_abs z).symm
   mul_im_I_ax _ := mul_one _
+  le := fun w z => w.re ≤ z.re ∧ w.im = z.im
+  le_refl := by simp only [le_refl, and_self, forall_const]
+  le_trans := fun _ _ _ h1 h2 => ⟨le_trans h1.1 h2.1, Eq.trans h1.2 h2.2 ⟩
+  le_antisymm := fun _ _ hab hba => Complex.ext (le_antisymm hab.1 hba.1) hab.2
+  le_iff_re_im := by simp only [not_and, AddMonoidHom.coe_mk, ZeroHom.coe_mk, forall_const]
```

Comment:
Isn't the proof of this `iff.rfl`?

Reply:
Which one do you mean? I assume you are takling about `le_iff_re_im`. In that case no it is not if.rfl

Reply:
Eric had a typo, `Iff.rfl` (note the capital `I`) works to prove `le_iff_re_im`